backend/controllers/container.rb

```diff
@@ -80,4 +80,26 @@ class ArchivesSpaceService < Sinatra::Base
   end
 
 
+  Endpoint.post('/repositories/:repo_id/top_containers_batch')
+    .description("Update a batch of yale containers")
+    .params(["ids", [Integer]],
+            ["ils_holding_id", String, "Value to set for ils_holding_id", :optional => true],
+            ["ils_item_id", String, "Value to set for ils_item_id", :optional => true],
+            ["repo_id", :repo_id])
+    .permissions([:manage_container])
+    .returns([200, :updated]) \
+  do
+    fields = {}
+    [:ils_holding_id, :ils_item_id].each do |fld|
```

Comment:
I wonder if it's worth creating a JSONModel to model the values that can be batch-updated -- it would save us having to cleanse the fields.